src/library/scala/collection/immutable/HashMap.scala

```diff
@@ -712,6 +970,41 @@ private final class BitmapIndexedMapNode[K, +V](
     )
   }
 
+  /** Variant of `copyAndMigrateFromNodeToInline` which mutates `this` rather than returning a new node.
+    *
+    * @param bitpos the bit position of the node to migrate inline
+    * @param oldNode the node currently stored at position `bitpos`
+    * @param node the node containing the single key-value pair to migrate inline
+    */
+  def migrateFromNodeToInlineInPlace[V1 >: V](bitpos: Int, oldNode: MapNode[K, V1], node: MapNode[K, V1]): Unit = {
```

Comment:
This functionality is now duplicated here and in the builder, probably best to use this method in the builder and remove that version.